wolfcrypt/src/asn.c

```diff
@@ -69,6 +69,10 @@ ASN Options:
 #include <wolfssl/wolfcrypt/wc_encrypt.h>
 #include <wolfssl/wolfcrypt/logging.h>
 
+#ifdef OPENSSL_EXTRA
+#include <wolfssl/openssl/ssl.h>
```

Comment:
It's a mistake to bring in ssl headers down into our crypto layer.  Especially as we're trying to completely separate them.